### ISSUE TITLE
Localizations: inaccurate translations for "expunge"

### ISSUE THREAD

**Issue by codegain** [roundcube/roundcubemail#8440]
Hi,

in the folder list, there is an option to "compact" the folder but by closer inspection it seems to execute the "expunge" imap command. I think the wording "Compact" is very confusing here, because it does not "compact" the folder in any way (like reduce file sizes of mails) but only deletes mails which are marked for deletion (if the setting to mark emails as deleted is turned on in the settings). (Please correct me if I'm wrong about the "expunge" command)

The german translation is "Packen" which is synonymous for "Compress", like a .zip file. The success message is also not helpful: "Der Ordner wurde erfolgreich gepackt" which is synonymous for "The folder was compressed sucessfully", but that is not what the command does. Looking at some other localizations, this might also be an issue there.

Could this be renamed to something like "Permanently delete messages marked for deletion"? I think this should clear up some confusion about this action.

Using Roundcube 1.4.13

**Comment by alecpl** [comment]
I agree it's not perfect, but I don't have a better idea. "expunge" and "purge" might also be not clear, and confusing especially with the "empty" action below in the menu. I think the icon (in Elastic) helps here a little bit. "Permanently delete messages marked for deletion" is too long.

ps. the Polish localization uses a word that translate to "to clean up/to organize".

**Comment by codegain** [comment]
@alecpl I don't exactly know the inner workings of IMAP, but today I also noticed that the "expunge" menu item is shown even if "Mark as deleted instead of moving to trash" is deactivated. Maybe this should then also be hidden if the setting is turned off (as the command does nothing it seems).
